ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -387,6 +387,20 @@ interface {{ ethernet_interface.name }}
 {%         if ethernet_interface.mac_access_group_out is arista.avd.defined %}
    mac access-group {{ ethernet_interface.mac_access_group_out }} out
 {%         endif %}
+{%         if ethernet_interface.multicast is arista.avd.defined %}
+{%             if ethernet_interface.multicast.ipv4.boundary is arista.avd.defined %}
+   multicast ipv4 boundary {{ ethernet_interface.multicast.ipv4.boundary }}
+{%             endif %}
+{%             if ethernet_interface.multicast.ipv6.boundary is arista.avd.defined %}
+   multicast ipv6 boundary {{ ethernet_interface.multicast.ipv6.boundary }} out
```

Comment:
Why is there out for IPv6 and not IPv4? (though granted `out` is mandatory for IPv6 in the version I checked 4.27.3)

```
DC1-LEAF1A(config-if-Et1)#multicast ipv4 boundary 1.1.1.1 ?
  out   only apply to outgoing traffic
  <cr>
```

Reply:
Worth to ask. 4.28.1:
```
POD1_LEAF01A(config-if-Et7)#multicast ipv4 boundary test
! Acl test not configured. Assigning anyway.
POD1_LEAF01A(config-if-Et7)#multicast ipv6 boundary test
% Incomplete command
```